recipe/meta.yaml

```
@@ -1,17 +1,17 @@
 {% set name = "htcondor" %}
-{% set version = "9_3_0" %}
+{% set version = "9.3.0" %}
 
 package:
   # the top-level package should be called `htcondor`, but
   # because the metapackage needs to specify host dependencies
   # we have to rename the top-level package so that conda-build
   # doesn't get confused
   name: {{ name|lower }}-build
-  version: {{ version|replace('_', '.') }}
+  version: {{ version }}
 
 source:
-  url: https://github.com/{{ name }}/{{ name }}/archive/V{{ version }}.tar.gz
-  sha256: 18e319d0cc827d0678b5fb5701c00b8e78c9e26ed89be9cc06b19b36bbb2bed5
+  url: https://research.cs.wisc.edu/htcondor/tarball/current/{{ version }}/release/condor-{{ version }}-src.tar.gz
```

```suggestion
  url: https://research.cs.wisc.edu/htcondor/tarball/feature/{{ version }}/release/condor-{{ version }}-src.tar.gz
```
Moving from calling "current" releases to "feature" releases. Tim has set up a symlink to point to the right place.